### ISSUE TITLE
Results from Browser History are not appearing in the search results

### ISSUE THREAD

**Issue by pavinduLakshan** [alyssaxuu/omni#19]
First off, congrats on the launch and thanks for making this extension. It is quite useful to search browser-related everything (tabs, history, bookmarks) with a single shortcut and without using a mouse.

That said, I have a bit problem that browser history can't be searched on Omni. 

For example, I searched for "manhole". It is showing in my browser history as follows.
![image](https://user-images.githubusercontent.com/41533942/149524223-6cf1c6f0-35ee-4db2-9df9-43b6066a00bc.png)

However, it is not showing in the omni search results.
![image](https://user-images.githubusercontent.com/41533942/149524351-a0372d64-9ac2-414b-9d62-50199895f9a4.png)

It was the same when I searched using terms "plan", "Business", "Xcore". Do you know why it occurs?   

My environment:  

Browser: Brave (Version 1.34.80 Chromium: 97.0.4692.71 (Official Build) (64-bit))
OS; Windows 10 education

Thanks in advance.

**Comment by alyssaxuu** [comment]
I wasn't sure whether to include the history results by default. I made it searchable using the command only to avoid showing any sensitive data. Some users have raised concerns about the extension even having access to the browser history to begin with (for privacy reasons), so I am not sure how to feel about it.

Open to changing it though, unsure what people think!

**Comment by Explosion-Scratch** [comment]
> 

You could make some sort of dropdown on the left of the search bar, then when the user tries to activate history ask for an optional permission

**Comment by alyssaxuu** [comment]
Yeah, could work. I think ideally I'd like to have a way for users to decide what they want to see, so they can customize if they see bookmarks, history, actions... and even in which specific order they want it to be. Not sure where that could go though, it would have to be prompted after install and accessible somewhere, probably as an action in the omnisearch, but maybe also through a popup by clicking on the icon in the toolbar, not sure.

**Comment by vladstudio** [comment]
Maybe: ask for history permission, if granted – include history into search results?

**Comment by wll8** [comment]
Try typing `/history` before searching:
![image](https://user-images.githubusercontent.com/28223260/154599643-b33741f4-fdb2-4709-a8d8-3719fd52f2ce.png)

**Comment by wll8** [comment]
Also, maybe the previous code only searched from nearly [1000 history records](https://github.com/alyssaxuu/omni/pull/70/commits/1523535bb66677f652d6e4956fe77eb66a3078f6). This should be a trade-off from the performance aspect, but the [PR#70](https://github.com/alyssaxuu/omni/pull/70) should solve this problem, quickly search from all records, whether it is 20,000 or more Two hundred thousand.

**Comment by gal-checksum** [comment]
+1 on having history searched directly from the search bar.
